frontend/src/modules/integration/components/integration-list-page.vue

```diff
@@ -99,13 +99,19 @@
   </app-page-wrapper>
 </template>
 
+<script setup>
+import { CrowdIntegrations } from '@/integrations/integrations-config';
+const supportUrl = CrowdIntegrations.getConfig('lfx').supportUrl || 'https://github.com/CrowdDotDev/crowd.dev/issues?q=is%3Aopen+is%3Aissue+label%3A%22Integration+%F0%9F%94%8C%22+';
```

Comment:
I think this is overcomplicated. Please dont use integration config if its not intended to be an actual integration. 
I would just change url in line 13 and thats it.

Reply:
done